redash/destinations/discord.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+import requests
+
+from redash.models import Alert
+from redash.destinations import BaseDestination, register
+from redash.utils import json_dumps
+
+colors = {
+    # Colors are in a Decimal format as Discord requires them to be Decimals for embeds
+    Alert.OK_STATE: "2600544",  # Green Decimal Code
```

Comment:
Python can read color code like `green or red`. So you can replace "260054" into `green`

Reply:
> It doesn't look like that's the case, just tested it and got the following worker log:
> 
> `[2023-07-11 08:58:58,920][PID:129][ERROR][root] Discord send ERROR. status_code => 400`
> 
> So looks like it has to be decimal codes rather than more straightforward codes like green and red.

You can change this code in your way. Let's try it again.

Reply:
Could you elaborate?